### PR TITLE
Start function clarifications

### LINKED ISSUE #896: Can start functions be imports?
The [current wording](https://github.com/WebAssembly/design/blob/master/Modules.md#module-start-function) isn't so clear about this:

> If a module contains a start node, the function must be defined in the module

I want to clarify some of the other wording in this entire section, but this question merits a bit more thought I think.

### PR DESCRIPTION
**Pull request by jfbastien**
Resolves #896.

### CODE REVIEW

File: Modules.md [hunk 1]
```diff
@@ -175,12 +175,12 @@ to allow *explicitly* sharing state between module instances.
 ## Module start function
 
 If the module has a start node defined, the function it refers should be called
-by the loader after the instance is initialized and before the exported functions
-are called.
+by the loader after the instance is initialized, including its Data and Element
```
Comment (rossberg-chromium) [inline Modules.md]: Wording nit: sections don't get initialised, memories and tables do (from them).

**Comment by rossberg-chromium** [comment]
LGTM modulo nit

File: Modules.md (hunk 1 above)
Comment (jfbastien) [inline Modules.md]: Right. Clarified, ty.

**Comment by jfbastien** [comment]
We don't specify that this happens before any instance-creating promise returns because that's JS.md stuff. I changed the phrasing a bit to say "before exported functions are callable", that's probably good enough to clarify the promise thing?

**Comment by rossberg-chromium** [comment]
SGTM. the original wording was weird, this is better.

**Pull request merged**